packages/@sanity/form-builder/src/inputs/files/common/FileInputButton/styles.ts

```diff
@@ -0,0 +1,37 @@
+import {Button, Theme} from '@sanity/ui'
+import styled, {css} from 'styled-components'
+import {focusRingStyle} from '../../../../components/withFocusRing/helpers'
+
+const buttonTheme = {border: {width: 1}}
```

Comment:
Any particular reason to have this as a nested object and not just a number value?

Reply:
Because this object will be implemented in the sanity ui theme, so it will be `theme.sanity.button.width`. But since it’s not implemented yet, I will replace with a value as you suggest 👍